packages/edit-site/src/components/block-editor/style.scss

```diff
@@ -1,29 +1,3 @@
-// The button element easily inherits styles that are meant for the editor style.
-// These rules enhance the specificity to reduce that inheritance.
-// This is duplicated in visual-editor.
-.edit-site-block-editor__editor-styles-wrapper  .components-button {
-	font-family: $default-font;
-	font-size: $default-font-size;
-	padding: 6px 12px;
-
-	&.is-tertiary,
-	&.has-icon {
-		padding: 6px;
-	}
-}
-
-.edit-site-visual-editor {
```

Comment:
This style moved to the specific editor-canvas-container component. It's only useful there.